<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="labelMonth">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string notr="true">&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;September 2009&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menu_Month">
    <property name="title">
     <string>&amp;Month</string>
    </property>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="actionCurrent"/>
    <addaction name="separator"/>
    <addaction name="actionMemo"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Month"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="MiniAppCalendar.qrc">
     <normaloff>:/images/images/open.gif</normaloff>:/images/images/open.gif</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open an existing file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="MiniAppCalendar.qrc">
     <normaloff>:/images/images/save.gif</normaloff>:/images/images/save.gif</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="statusTip">
    <string>Save the document to disk</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="statusTip">
    <string>Save the document under a new name</string>
   </property>
  </action>
  <action name="actionNew_File">
   <property name="icon">
    <iconset resource="MiniAppCalendar.qrc">
     <normaloff>:/images/images/new.gif</normaloff>:/images/images/new.gif</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
   <property name="statusTip">
    <string>Create a new document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset resource="MiniAppCalendar.qrc">
     <normaloff>:/images/images/prev.gif</normaloff>:/images/images/prev.gif</iconset>
   </property>
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="statusTip">
    <string>Go to the previous month</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset resource="MiniAppCalendar.qrc">
     <normaloff>:/images/images/next.gif</normaloff>:/images/images/next.gif</iconset>
   </property>
   <property name="text">
    <string>Next</string>
   </property>
   <property name="statusTip">
    <string>Go to the next month</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionCurrent">
   <property name="icon">
    <iconset resource="MiniAppCalendar.qrc">
     <normaloff>:/images/images/current.gif</normaloff>:/images/images/current.gif</iconset>
   </property>
   <property name="text">
    <string>Current</string>
   </property>
   <property name="statusTip">
    <string>Go to the current month</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionMemo">
   <property name="icon">
    <iconset resource="MiniAppCalendar.qrc">
     <normaloff>:/images/images/memo.gif</normaloff>:/images/images/memo.gif</iconset>
   </property>
   <property name="text">
    <string>Memo</string>
   </property>
   <property name="statusTip">
    <string>Display the monthly memo</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="statusTip">
    <string>About Qt</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutQtRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MiniAppCalendar.qrc"/>
 </resources>
 <connections/>
</ui>
